src/mongocxx/options/find_one_and_update.hpp

```diff
@@ -27,6 +26,17 @@ MONGOCXX_INLINE_NAMESPACE_BEGIN
 namespace options {
 
 ///
+/// Argument for find_one_and_update and find_one_and_replace to determine
+/// whether to return the document before or after modification.
+///
+/// @see https://docs.mongodb.org/manual/reference/method/db.collection.findAndModify/#return-data
+///
+enum class return_document {
```

Comment:
I think it would be better if this lived in its own file, logically it's weird that either find_one_and_* class would have to rely on the other's header.